UITests/UITests.App/UITests.App.csproj

```diff
@@ -133,6 +133,12 @@
     <Compile Include="MainPage.xaml.cs">
       <DependentUpon>MainPage.xaml</DependentUpon>
     </Compile>
+    <Compile Include="Pages\SimpleTest.xaml.cs">
```

Comment:
@azchohfi can we use the SDK Extras to switch to the new-style project here? Don't want to get into merge problems in the future with multiple-tests being written.

Otherwise we can try using the glob syntax instead?